<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(47, 30, 111);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_username">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 700 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_password">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 700 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_signup">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(137, 174, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Admin Signup</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_professor_id">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Professor ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_professor_id">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 700 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
